### PR TITLE
Ux/inspector rearrange

### PR DESCRIPTION
**Pull request by OmarDaSilva**
Rearranged the inspector

Problem:
The inspector panel had an undesirable arrangement and needed to be rearranged

FIx:
In inspector.tx where it returns the components of the inspector to be rendered (Line 379). I rearranged the components by swapping them to different lines, e.g. ClassNameSubsection originally near the bottom, however, moved to the top.

### CODE REVIEW

**Review by seanparsons — APPROVED** [review]

**Comment by OmarDaSilva** [comment]
This PR is ready to be merged, could I get approvers or more comments, please?

File: editor/src/components/inspector/inspector.tsx [hunk 1]
```diff
@@ -413,13 +404,21 @@ export const Inspector = betterReactMemo<InspectorProps>('Inspector', (props: In
...
+            onWrap={props.onWrap}
+            onUnwrap={props.onUnwrap}
+            value={props.input.layoutWrapper}
+          />
         </React.Fragment>
       )
     }
   }
-
+  //first
```
Comment (balazsbajorics) [inline editor/src/components/inspector/inspector.tsx:421]: I'd remove this comment from here

```suggestion
```

File: editor/src/components/inspector/sections/style-section/style-section.tsx [hunk 2]
```diff
@@ -8,6 +8,14 @@ import { ShadowSubsection } from './shadow-subsection/shadow-subsection'
 import { TextShadowSubsection } from './text-subsection/text-shadow-subsection'
 import { AutosizingTextSubsection } from './text-subsection/text-subsection'
 import { TransformSubsection } from './transform-subsection/transform-subsection'
+import {
+  FlexRow,
```
Comment (enidemi) [inline editor/src/components/inspector/sections/style-section/style-section.tsx:12]: these imports seem unused, can you please delete them? :)

File: editor/src/components/inspector/sections/event-handlers-section/event-handlers-section.tsx [hunk 3]
```diff
@@ -57,7 +57,7 @@ export const EventHandlersSection = betterReactMemo('EventHandlersSection', () =
   }
 
   return (
-    <>
+    <React.Fragment>
```
Comment (balazsbajorics) [inline editor/src/components/inspector/sections/event-handlers-section/event-handlers-section.tsx:60]: `<>` is actually a shorthand version to write `<React.Fragment>`

in some places in the codebase we use `<React.Fragment>` if we also use Emotion's css pragma (how to see if the pragma is used: the first line of a file will contain `/** @jsx jsx */` ). In other files, it is not necessary to write React.Fragment, you can use the `<></>` shorthand too.

File: editor/src/components/inspector/sections/style-section/style-section.tsx [hunk 4]
```diff
@@ -8,6 +8,14 @@ import { ShadowSubsection } from './shadow-subsection/shadow-subsection'
...
+  InspectorSectionHeader,
+  InspectorSubsectionHeader,
+  Section,
+} from '../../../../uuiui'
+import { GridRow } from '../../widgets/grid-row'
+import { InspectorContextMenuWrapper } from '../../../../uuiui-deps'
```
Comment (balazsbajorics) [inline editor/src/components/inspector/sections/style-section/style-section.tsx:18]: it seems like all these new imports were not used in the end, please remove them

**Review by enidemi — APPROVED** [review]

**Review by balazsbajorics — CHANGES_REQUESTED** [review]
please remove that leftover comment and unnecessary imports, otherwise looks good! :)

File: editor/src/components/inspector/inspector.tsx (hunk 1 above)
Comment (OmarDaSilva) [reply editor/src/components/inspector/inspector.tsx:421]: Done!

**Review by balazsbajorics — APPROVED** [review]

**Pull request merged**